pyproject.toml

```diff
@@ -15,19 +15,19 @@ classifiers = [
     "Intended Audience :: Developers",
     "License :: OSI Approved :: Apache Software License",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.7",
-    "Programming Language :: Python :: 3.8",
```

Comment:
why dropping python 3.8? We should support it.

Reply:
I got this error when using 3.8. It works with Python 3.8.1.
```
The current project's Python requirement (>=3.8,<4.0) is not compatible with some of the required packages Python requirement:
    - langchain requires Python >=3.8.1,<4.0, so it will not be satisfied for Python >=3.8,<3.8.1
```